crates/electrum/tests/test_electrum.rs

```diff
@@ -19,6 +21,222 @@ fn get_balance(
     Ok(balance)
 }
 
+#[test]
+pub fn test_update_tx_graph_without_keychain() -> anyhow::Result<()> {
```

Comment:
📌 If I recall, the name of this test is a relic from before the `sync` and `full_scan` concepts were developed. I might now call it `sync_should_update_chain_and_tx_graph` or similar, but not terribly important at the moment.